posthog-android/src/main/java/com/posthog/android/replay/PostHogSessionReplayConfig.kt

```diff
@@ -36,4 +38,12 @@ public class PostHogSessionReplayConfig(
      */
     @PostHogExperimental
     public var screenshot: Boolean = false,
+    /**
+     * Deboucer delay used to reduce the number of snapshots captured and reduce performance impact
+     * This is used for capturing the view as a wireframe or screenshot
+     * The lower the number more snapshots will be captured but higher the performance impact
+     * Defaults to 500ms
+     */
+    @PostHogExperimental
+    public var debouncerDelayMs: Long = 500,
```

Comment:
could nitpick and suggest we call this frames per second (so debounce delay of  500ms === an FPS of 2)

Reply:
but no strong opinion